app/config/collections.php

```diff
@@ -2099,7 +2099,18 @@
                 'required' => false,
                 'default' => null,
                 'filters' => [],
-            ]
+            ],
+            [
+                '$id' => ID::custom('apiTokenScopes'),
```

Comment:
This doesn't really fit well with the naming style of other attributes we have in this collection and others. We would usually try to avoid namespaces as much as possible and call something like this simply `scopes`.